java-api-core/src/com/kinvey/java/model/KinveySyncSaveBatchResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.kinvey.java.model
+
+import com.google.api.client.json.GenericJson
+import com.google.api.client.util.Key
+
+class KinveySyncSaveBatchResponse<T>(
+    var entityList: Collection<T?>?,
+    var errors: List<KinveyBatchInsertError>?) : GenericJson()
```

Comment:
this one extends from `GenericJson` for example. I'm just trying to understand when we need and when we don't need to extends from `GenericJson`

Reply:
Yes, I used extending from `GenericJson` for response classes in my last implementation of methods for multi-insert in `SyncManager`. Now it's none needed. Removed.